jobs/datadog-consul/spec

```diff
@@ -0,0 +1,12 @@
+---
+name: datadog-consul
+packages: []
+
+templates:
+  process.yaml.erb: config/datadog-integrations/process.yaml
+  tcp_check.yaml.erb: config/datadog-integrations/tcp_check.yaml
+
+properties:
+  consul.port:
+    description: "Listening Port for consul agent."
```

Comment:
Well, 
> This script, `datadog-consul` is listening for any traffic on the consul with given port, in order to establish if consul is not refusing the connections.

Not sure, if that exact statement needs to be in the code, as:
1. It's part of the script, so we know who.
1. Port is provided below, and the script is doing a tcp check.
1. I'm not sure, if `why`, should be a question here.

I'm happy to change it. Perhaps you could help me to gather correct words? :)